src/View/ImageFusion/UITransferROIWindow.py

```diff
@@ -440,91 +447,116 @@ def save_clicked(self, interrupt_flag, progress_callback):
         :param progress_callback: signal that receives the current
                                   progress of the loading.
         """
-        progress_callback.emit(("Converting images to sitk", 0))
+        try:
+            progress_callback.emit(("Converting images to sitk", 0))
```

Comment:
this message is useful to the developer for debugging.  What would be useful for a non-technical user?